src/lib/y2partitioner/widgets/menus/add.rb

```diff
@@ -27,6 +27,10 @@ module Widgets
     module Menus
       # Class to represent the Add menu
       class Add < Device
+        extend Yast::I18n
```

Comment:
Why extend?

Reply:
As already discussed offline, because the @joseivanlopez's suggestion.